packages/amplify-frontend-android/lib/amplify-config-helper.js

```diff
@@ -10,6 +10,7 @@ function generateConfig(context, newAWSConfig) {
   constructAuth(metadata, amplifyConfig, newAWSConfig);
   constructPredictions(metadata, amplifyConfig);
   constructStorage(metadata, amplifyConfig);
+  constructGeo(metadata, amplifyConfig);
```

Comment:
https://github.com/aws-amplify/amplify-cli/pull/8031/files#diff-90dee687551f26638056d4272cdc090ec8a03f740aad8700ba8b79060c724bcaR10 and https://github.com/aws-amplify/amplify-cli/pull/8890/files#diff-90dee687551f26638056d4272cdc090ec8a03f740aad8700ba8b79060c724bcaR191